### PR TITLE
It's still failing, but more of the tests work

### PR DESCRIPTION
**Pull request by wallymathieu**
I added some test files (man pages) from ubuntu. In order to use the test files i did:

```
ManParser.use_filesystem(FakeFileSystem.new({
  'ls' => $ls_man, 
  'printf' => $printf_man, 
  'grep' => $grep_man, 
  'xargs'=> $xargs_man
}))
```

where FakeFileSystem is defined in the spec helper

### CODE REVIEW

File: man_parser.gemspec [hunk 1]
```diff
@@ -18,21 +18,15 @@ Gem::Specification.new do |s|
...
      "Rakefile",
      "VERSION",
-     "lib/man_parser.rb",
      "man_parser.gemspec",
-     "spec/call",
-     "spec/man_parser_spec.rb",
-     "spec/spec_helper.rb"
-  ]
+     "spec/call"
+  ]+Dir.glob('lib/**/*.rb')+Dir.glob('spec/**/*.rb')+Dir.glob('spec/**/*.txt')
```
Comment (grosser) [inline man_parser.gemspec:23]: no need to ship spec files

**Comment by grosser** [comment]
yeah this is going into really weird places ... maybe just test on linux ? or is there a good usecase for osx ?

**Comment by wallymathieu** [comment]
Yea, probably not.

**Comment by wallymathieu** [comment]
I'm closing the pull request. I picked the man files from ubuntu, but it looks as if they have changed since this lib was written (that's why I thought it would be a good idea to include them as test files).

**Pull request closed without merging**